recipes/certifi/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: certifi
+  version: {{ version }}
+
+source:
+  fn: certifi-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/c/certifi/certifi-{{ version }}.tar.gz
+  sha256: 1
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - certifi
+
+about:
+  home: http://certifi.io/
+  license: ISC
+  summary: Python package for providing Mozilla's CA Bundle.
+
+extra:
+  recipe-maintainers:
+    - dan-blanchard
```

Comment:
Hmm...interesting. I added an issue ( https://github.com/conda-forge/conda-forge.github.io/issues/84 ) so we can track this.